tests/test_database.py

```
@@ -672,3 +672,134 @@ def test_update_issue_assignment_rejects_invalid_new_worker(mock_get_client):
+            "status": "completed",
+        }
+    ]
+    mock_eq.execute.return_value = mock_execute
+    mock_get_client.return_value = mock_client
+
+    # Should not raise
+    update_patch_status(1, "completed")
+
+
+@patch("rouge.core.database.get_client")
+def test_update_patch_status_invalid_status(mock_get_client):
```

Ruff (0.14.11)

Unused function argument: `mock_get_client`

(ARG001)